src/server/pfs/server/driver.go

```diff
@@ -853,6 +861,11 @@ nextSubvBranch:
 			// provBranchInfo.HEAD's provenance. Every commit in there will be the
 			// HEAD of some other provBranchInfo.
 		}
+		if head != nil {
```

Comment:
I think the comment above this is now inaccurate.

Reply:
I'm not sure it is inaccurate now, we're still not inspecting `provBranchInfo.HEAD`'s provenance, we're inspecting `branchInfo.HEAD`'s provenance. I did change the comment a little bit because I think it's not the `propagateBranch` invariant so much as the fact the provenance is stored as a transitive closure.